scalr/README.md

```diff
@@ -0,0 +1,71 @@
+# Scalr Integration
+
+## Overview
+
+This integration collects metrics from Scalr.
+
+## Setup
+The Scalr integration is not included in the [Datadog Agent][2] package, so you need to install it.
+
+### Installation
+
+For Agent v7.21+ / v6.21+, follow the instructions below to install the Scalr integration on your host. See [Use Community Integrations][3] to install with the Docker Agent or earlier versions of the Agent.
+
+1. Run the following command to install the Agent integration:
+
+   ```shell
+   datadog-agent integration install -t datadog-scalr==1.0.0
+   ```
+
+2. Configure your integration similar to Agent-based [integrations][4].
+
+### Configuration
+
+1. Edit the `scalr.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][7] to start collecting your Scalr [metrics](#metrics). See the [sample syncthing.d/conf.yaml][8] for all available configuration options.
+
+2. [Restart the Agent][9]
+
+### Validation
+
+Run the [Agent's status subcommand][10] and look for `scalr` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][11] for a list of metrics provided by this check.
```

Comment:
This is just a formatting thing (check is fine but to generate the table, we need to use integration).

```suggestion
See [metadata.csv][11] for a list of metrics provided by this integration.
```